### ISSUE TITLE
PCR duplication

### ISSUE THREAD

**Issue by dzaccook** [schatzlab/genomescope#37]
Hi,
Should I remove the PCR duplicated reads before running jellyfish?
Thank you!

Again, after remove PCR duplicated reads, I run jellyfish and GS2, and get the results,
![transformed_linear_plot](https://user-images.githubusercontent.com/23490134/84845456-48fb8600-b07f-11ea-84ca-46aba79b070f.png)
The estimated genome size is about 1.15Gb.
On the other hand, I plot the jellyfish histo result, and stat the total number of Kmers,  I got a genome size=Total number of Kmers / Pkdepth=2.1Gb.
![image](https://user-images.githubusercontent.com/23490134/84846041-80b6fd80-b080-11ea-858c-ace8cabb67fb.png)
Which was right?
Thanks a lot!

**Comment by tbenavi1** [comment]
No, it is generally not necessary to remove PCR duplicated reads except perhaps for extreme levels. GenomeScope uses a negative binomial model rather than a strict Poisson to account for the overdispersion that occurs from PCR duplicates. 

As for the genome size estimate, GenomeScope calculated its results based on a pkdepth of 53.9 (kcov in the plot you provided). As you can see, there is a slight peak located at 53.9, which is why GenomeScope focused in on it. For some data this would be the correct peak, but for other data it may not be. I would first make sure you are running GenomeScope version 2.0 (https://github.com/tbenavi1/genomescope2.0). Then, to tell GenomeScope to treat the peak at 117 as the first peak, you can run with the additional parameter "-l 117". 

Let me know if you have any further questions. Thanks.

**Comment by dzaccook** [comment]
Thank you! I understand.

**Comment by dzaccook** [comment]
Hi,
Can GenomeScope 2.0 output the total number of Kmers? Is it different with the "jellyfish stats" function? 
Like above, if I  run GenomeScope 2.0 with the additional parameter "-l 117", the genome size will be about a half of the result with default parameters  (based on a pkdepth of 53.9). The difference is too large with the jellyfish result.  I thought that surely there had been a mistake when I calculated it. Can you give me some help?
Thank you!
